src/node_http_parser.cc

```diff
@@ -136,7 +136,10 @@ struct StringPtr {
 
   Local<String> ToString(Environment* env) const {
     if (str_)
-      return OneByteString(env->isolate(), str_, size_);
+      return String::NewFromUtf8(env->isolate(),
```

Comment:
The benchmarks would be bias in out current benchmarks. We need to set some up that have various header lengths.